examples/count/main.cc

```diff
@@ -6,43 +6,43 @@ using namespace reactor;
 using namespace std::chrono_literals;
 
 class Count : public Reactor {
- private:
-  // actions
-  Timer timer{"timer", this};
-  LogicalAction<int> counter{"counter", this};
-
-  // reactions
-  Reaction r_init{"r_init", 1, this, [this]() { init(); }};
-  Reaction r_counter{"r_counter", 2, this, [this]() { print_count(); }};
-
- public:
-  Count(Environment* env) : Reactor("Count", env) {}
-
-  void assemble() override {
-    r_init.declare_trigger(&timer);
-    r_counter.declare_trigger(&counter);
-  }
-
-  void print_count() {
-    auto& value = *(counter.get());
-    std::cout << "Count: " << value << std::endl;
-    counter.schedule(value + 1, 1s);
-  }
-
-  void init() {
-    std::cout << "Hello World!" << std::endl;
-    counter.schedule(0, 1s);
-  }
+private:
```

Comment:
doesn't look like this file has been formatted with the current .clang-forma (it used 4 spaces for indent, while 2 spaces are configured). Can we enforce correct formatting by running clang-format in CI and error out if it would do any reformatting?